### ISSUE TITLE
Validation error response

### ISSUE THREAD

**Issue by david4096** [ga4gh/server#804]
When receiving a `400 Bad Request` it would be helpful if the fields in the request that didn't pass validation were in the message. For example, if a string is sent in place of an integer.

**Comment by diekhans** [comment]
How the protocol buffers parser behaves will be a factor in what
error information can be returned.  Work on this issue should
be delayed until after the conversion.

**Comment by jeromekelleher** [comment]
Agreed with @diekhans --- we won't know quite how to do this until we've finished the protobuf conversion.
